src/lib.rs

```diff
@@ -1,5 +1,5 @@
 use ::tantivy as tv;
-use ::tantivy::schema::{Term, Value};
+use ::tantivy::schema::{Term, OwnedValue as Value};
```

Comment:
Reminder remove alias

Reply:
I'm not going to change these alias in this PR. If we want to change it, it can come in a future PR.